extensions/wikia/PortableInfobox/services/Helpers/ImageFilenameSanitizer.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Wikia\PortableInfobox\Helpers;
+
+class ImageFilenameSanitizer {
```

Comment:
It looks like a general-interest functionality. Maybe we have an existing implementation somewhere else or we should put it in some common place?